sourcefiles/randosettings.py

```diff
@@ -101,6 +101,8 @@ class GameFlags(SerializableFlag):
     CHRONOSANITY = auto()
     ROCKSANITY = auto()
     TAB_TREASURES = auto()  # Maybe needs to be part of treasure page?
+    TAB_RANDO = auto()
```

Comment:
Are these going to be added to the GUI, or are they supposed to be web-generator only?

Reply:
Shouldn't be a problem adding them to the GUI

Reply:
Added.   Here is what the first cut looks like

<img width="502" height="690" alt="image" src="https://github.com/user-attachments/assets/c45c94b0-d16a-4acc-8b65-74dd4cebd07c" />

Reply:
Good idea,  they are mutually exclusive and this a great way to handle it.  Thanks,  I’ll make the update tonight

Reply:
Updated.  Here is what it looks like:

<img width="166" height="169" alt="image" src="https://github.com/user-attachments/assets/139c3a16-e894-4959-8524-cf1b0e2ad8fd" />